packages/shared/src/components/sidebar/sections/MainSection.tsx

```diff
@@ -25,6 +36,18 @@ export const MainSection = ({
         }
       : undefined;
 
+    const plus = !isPlus
```

Comment:
@omBratteng This will show it to everyone regardless of experiment enrollment right?

Reply:
Added seperate PR: https://github.com/dailydotdev/apps/pull/3844